matrimony_admin/views.py

```
@@ -161,8 +168,94 @@ def get_context_data(self, **kwargs):
+class DeleteUserView(View):
+    success_url = reverse_lazy('user_manage')  # Redirect after successful deletion
+
+    def post(self, request, *args, **kwargs):
+        # Get the list of selected users from POST data
+        selected_users = request.POST.getlist('selected_users')
+
+        if selected_users:
+            # Delete all selected users
+            costume_user.objects.filter(id__in=selected_users).delete()
+
+        return HttpResponseRedirect(self.success_url)
```

Ruff

`HttpResponseRedirect` may be undefined, or defined from star imports

(F405)